knowledge_base/bundle_variables/README.md

```diff
@@ -0,0 +1,65 @@
+# bundle_variables
+
+Example to show how bundle variables work.
+
+We will focus on the behavior of the variables section alone. To do so, we exclusively use
+the `validate` command, and use its JSON output mode to focus on the variables section.
+
+## Usage
```

Comment:
This singular heading feels unnecessary and also doesn't seem like quite the right word. I'd either remove it (acceptable for a KB e.g., https://kb.databricks.com/en_US/clusters/calculate-number-of-cores) or break up the content into at least two headings. (I'd lean towards the former.)